src/test/java/org/candlepin/service/impl/DefaultUserServiceAdapterTest.java

```diff
@@ -210,115 +205,114 @@ public void testUpdating() {
         assertFalse(updated.isSuperAdmin());
     }
 
-
     // @Test
     // public void testGetAccessibleOwners() {
-    //     String username = "TESTUSER";
-    //     String password = "sekretpassword";
-    //     Owner owner1 = new Owner().setKey("owner1").setDisplayName("owner one");
-    //     Owner owner2 = new Owner().setKey("owner2").setDisplayName("owner two");
-    //     User user = new User(username, password);
-
-    //     Set<Owner> owners = user.getOwners(null, Access.ALL);
-    //     assertEquals(0, owners.size());
-    //     user.addPermissions(new TestPermission(owner1));
-    //     user.addPermissions(new TestPermission(owner2));
-
-    //     // Adding the new permissions should give us access
-    //     // to both new owners
-    //     owners = user.getOwners(null, Access.ALL);
-    //     assertEquals(2, owners.size());
+    // String username = "TESTUSER";
```

Comment:
Utterly destroyed comment block

Reply:
Tests should be disabled, not commented out.